EDDiscovery/UserControls/UserControlScanGrid.cs

```diff
@@ -226,10 +272,12 @@ public override void InitialDisplay()
 
                         DataGridViewRow cur = dataGridViewScangrid.Rows[dataGridViewScangrid.Rows.Count - 1];
 
-                        string scan = sn.ScanData.DisplayString(); // display tooltip with full information when hower bodies image and name
+                        string scan = sn.ScanData.DisplayString(); // display tooltip with full information when hower bodies image
                         cur.Cells[0].ToolTipText = scan;
-                        cur.Cells[1].ToolTipText = scan;
                         cur.Tag = img;
+
+                        // display total scan values
+                        BuildSystemInfo(last_sn);
```

Comment:
This build is in the wrong place, you don't do it ever node iteration. fixed on master

Reply:
Great, thanks to let me know and help in understand. ;)